docs/sources/tutorials/alerting-get-started-pt6/index.md

```diff
@@ -259,57 +259,47 @@ You have successfully linked this alert rule to your visualization!
+- **View panel**: links directly to the individual panel where the alert was triggered
 
-```
-{
-  "receiver": "prod-alerts",
-  "status": "firing",
-  "alerts": [
-    {
-      "status": "firing",
-      "labels": {
-        "alertname": "cpu-usage",
-        "deployment": "prod-us-cs30",
-        "grafana_folder": "sys-metrics",
-        "instance": "flask-prod:5000",
-        "job": "flask"
-      },
-      "annotations": {},
-      "silenceURL": "http://localhost:3000/alerting/silence/new?
-      "dashboardURL": "http://localhost:3000/d/dc203378-1ef9-410b-a636-b533a0dd3bd8?from=1748934450000&orgId=1&to=1748938080006",
-      "panelURL": "http://localhost:3000/d/dc203378-1ef9-410b-a636-b533a0dd3bd8?from=1748934450000&orgId=1&to=1748938080006&viewPanel=2",
+{{< figure src="/media/docs/alerting/email-notification-w-url.png" max-width="1200px" caption="Alert notification with links to panel and dashboard." >}}
 
-... }
-```
+Clicking either button opens Grafana with a pre-applied time range relevant to the alert.
 
-_Received alert notification in webhook Contact point_
+By default, this URL includes `from` and `to` query parameters that reflect the time window around the alert event. This helps you land directly in the time window where the alert occurred, making it easier to analyze what happened.
 
-It’s worth mentioning that alert rules that are linked to a panel include a link to said visualization in the alert notifications. In the alert notification example above, the message includes useful information such as the summary, description, and a link to the relevant dashboard for the firing or resolved alert (i.e. `dashboardURL`). This helps responders quickly navigate to the appropriate context for investigation.
-
-You can extend this functionality by adding a custom annotation to your alert rules and creating a notification template that [includes a link to a dashboard with a time range.](https://grafana.com/docs/grafana/latest/alerting/configure-notifications/template-notifications/examples/#print-a-link-to-a-dashboard-with-time-range). The URL will include a time range based on the alert’s timing—starting from one hour before the alert started (`from`) to either the alert’s end time or the current time (`to`), depending on whether the alert is resolved or still firing.
+If you want to define a more intentional time range, for example, one hour before and after the alert, you can customize your notifications using a [notification template](https://grafana.com/docs/grafana/latest/alerting/configure-notifications/template-notifications/examples/#print-a-link-to-a-dashboard-with-time-range). With a template, you can explicitly set `from` and `to` values for more precise control over what users see when they follow the dashboard link.
 
 The final URL is constructed using a custom annotation (e.g., `MyDashboardURL`) along with the `from` and `to` parameters, which are calculated in the notification template.
```

Comment:
suggest merging this within the previous paragraph which both mention the same example